packaging/Flatpak/org.telesculptor.TeleSculptor.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>org.telesculptor.TeleSculptor</id>
+    <launchable type="desktop-id">org.telesculptor.TeleSculptor.desktop</launchable>
+    <name>TeleSculptor</name>
+    <summary>Photogrammetry designed with a focus on aerial video</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>BSD-3-Clause</project_license>
+    <description>
+        <p>
+            TeleSculptor is a cross-platform desktop application for photogrammetry.
+            It was designed with a focus on aerial video, such as video collected from
+            UAVs, and handles geospatial coordinates and can make use of metadata,
+            if available, from GPS and IMU sensors. However, the software can also work
+            with non-geospatial data and with collections of images instead of metadata.
+            TeleSculptor uses Structure-from-Motion techniques to estimate camera
+            parameters as well as a sparse set of 3D landmarks.
+            It uses Multiview Stereo techniques to estimate dense depth maps on key
+            frame and then fuses those depth maps into a consistent surface mesh which
+            can be colored from the source imagery.
+        </p>
+    </description>
+    <url type="homepage">https://github.com/Kitware/TeleSculptor</url>
```

Comment:
It might be better for this to be `https://telesculptor.org/`, which is the new homepage we've launched recently.  It links back to the Github page.

Reply:
I will change it.